Formula/readtools@1.4.rb

```diff
@@ -1,11 +1,9 @@
-class Readtools < Formula
+class ReadtoolsAT14 < Formula
   desc "Handling Sequence Data from Different Sequencing Platforms"
   homepage "https://magicdgs.github.io/ReadTools/"
   url "https://github.com/magicDGS/ReadTools/releases/download/1.4.1/ReadTools.jar"
   sha256 "7092f3de1de6eb24023b1a73a0e362feebc98145ad07310ef5c0293fe2934635"
 
-  head { url "https://github.com/magicDGS/ReadTools.git" }
```

Comment:
Why removing the `head`? If that's a requirement for the `AT*` formulae, that should be added to the PR template for update formula. Also, if that's something that it's wrong, it should be fixed in other `ReadtoolsAT*` formulae.

Reply:
Because it seems that brew audit --strict does not like it. 

I guess it does not really matter otherwise, at least it does not give an annoying error message like the missing ...AT14 in the class name. I'm sorry for fixed this in arush, but I am currently installing/reinstalling a bunch of stuff that does not work with brew and this message was beginning to go on my nerves.

Reply:
Sure makes sense - so it should be added to the PR template and also should be changed in the other frozen versions. Maybe open an issue or a PR, otherwise I will do it one day of this week.